src/atf_model.ts

```diff
@@ -0,0 +1,17 @@
+import * as assert from 'assert';
+import { EOL } from 'os';
+
+/**
+ * Extract the project code from an ATF file.
+ *
+ * @param atfText The contents of the ATF file
+ * @returns The code of the Oracc project the file refers to
+ */
+export function getProjectCode(atfText: string): string {
+    const tag = "#project:";
+    const codes = atfText.split(EOL)
```

Comment:
What happens if there's no "#project" in the ATF? Maybe it can be mispelled, or have a space in the wrong place, or just be missing entirely.

Reply:
Good point. I've changed it to throw an error in that case, before the validation starts. Having a space (e.g. `#project :`) should throw the error too, I think (as it does now); in that case the line is a comment, or at least that's how we treat it for syntax highlighting.